toolkit/scripts/toolkit.mk

```diff
@@ -127,3 +127,9 @@ $(valid_arch_spec_names): $(go-specarchchecker) $(chroot_worker) $(local_specs)
 		--log-level=$(LOG_LEVEL) \
 		--log-file="$(valid_arch_spec_names_logs_path)" \
 		--log-color="$(LOG_COLOR)"
+
+##help:target:install-azurelinux-prereqs=Install Azure Linux Prerequisites
+install-azurelinux-prereqs:
```

Comment:
`azurelinux` is kind of an overloaded term in this context. Is it referring to the image being built or the build host?

Would it be possible to detect the OS being run on and then pick the correct installation method? If you don't want to implement Ubuntu, then you could error out.

Reply:
Can probably check /etc/os-release for that, I'll give it a shot